content/applications/sales/sales/products_prices/prices/pricing.rst

```diff
@@ -1,146 +1,468 @@
+to the amount calculated with the discount. After that, enter a desired figure in the
+:guilabel:`Rounding Method` field. The rounding method sets the price so that it is a multiple of
+the value in the field.
 
-In case of discount, you can show the public price and the computed discount % on printed sales
-orders and in your eCommerce catalog. To do so:
+.. note::
+   Rounding is applied *after* the discount and *before* the surcharge.
+
+.. tip::
+   To have prices that end in 9.99, set the :guilabel:`Rounding Method` to `10` and the
+   :guilabel:`Extra Fee` to `-0.01`.
+
+Lastly, specify the minimum amount of margin over the base price in the :guilabel:`Margins` field.
+
+Once all formula-related configurations are complete, Odoo provides an example of the formula in
+a blue block to the right of the configurations.
+
+.. example::
+   To apply 20% discounts, with prices rounded up to 9.99, set the :guilabel:`Discount` field to
+   `20`, the :guilabel:`Extra Fee` field to `-0.01`, and the :guilabel:`Rounding Method` field to
+   `10`.
+
+   .. image:: pricing/formula-discount-example.png
+      :align: center
+      :alt: Example of a 20% discount with prices rounded to 9.99 in Odoo Sales.
+
+Conditions
+----------
+
+At the bottom of the :guilabel:`Create Pricelist Rules` pop-up form is the :guilabel:`Conditions`
+section.
+
+Here, start by selecting one of the options in the :guilabel:`Apply On` field:
+
+- :guilabel:`All Products`: the advanced pricelist rule will be applied to all products.
+- :guilabel:`Product Category`: the advanced pricelist rule will be applied to a specific category
+  of products.
+- :guilabel:`Product`: the advanced pricelist rule will be applied to a specific product.
+- :guilabel:`Product Variant`: the advanced pricelist rule will be applied to a specific product
+  variant.
+
+If any of those options, apart from :guilabel:`All Products`, are selected, a new option-specific
+field appears, in which the specific :guilabel:`Product Category`, :guilabel:`Product`, or
+:guilabel:`Product Variant` must be chosen.
 
-* Check *Allow discounts on sales order lines* in :menuselection:`Sales --> Configuration -->
-  Settings --> Quotations & Sales --> Discounts`.
-* Apply the option in the pricelist setup form.
+Then, select a minimum quantity to be applied to the advanced pricelist rule in the :guilabel:`Min.
+Quantity` field. Lastly, select a range of dates for the pricelist item validation in the
+:guilabel:`Validity` field.
 
-.. image:: pricing/discount_options.png
+Once all configurations are complete, either click :guilabel:`Save & Close` to save the advanced
+pricelist rule, or click :guilabel:`Save & New` to immediately create another advanced pricelist
+rule on a fresh form.
+
+.. note::
+   If a price rule is set for a particular product, and another one for its product category, Odoo
+   takes the rule of the product itself. Make sure at least one pricelist item covers all products.
```

Comment:
I'm confused by this last sentence, "Make sure at least one pricelist item covers all products." I don't understand why that would be necessary. It's okay for pricelists to only include some particular products and not have a rule for all products. I think you could delete this sentence?